modules/vault-secrets.nix

```diff
@@ -115,6 +115,15 @@ let
           '';
         };
 
+        group = mkOption {
+          type = with types; nullOr str;
+          default = null;
+          example = "gitlab-runner";
+          description = ''
+            Group that should own the secrets files. Defaults to root.
```

Comment:
It doesn't though :thinking: 

If it was intended to have root group, then set `default = "root"`, otherwise set `default = "nobody"`

Reply:
The wonders of copy paste.